src/main/sensors/gyroanalyse.h

```diff
@@ -28,6 +28,11 @@
 // max for F3 targets
 #define FFT_WINDOW_SIZE 32
 
+typedef union gyroDynamicFilter_u {
```

Comment:
Unfortunately this is breaking the tests, since we can not include `gyroanalyse.h` in them due to `arm_math.h` not being available on x86. 

My suggestion on how to address this is to move `gyroDynamicFilter_t` into `gyro.h`, and then include this in here. (The include for `common/filter.h` will also have to be moved into `gyro.h`.)